draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -551,49 +550,47 @@ the raw public key by value in SD-JWT.
 
 
 
-## Combined Format for Issuance {#combined_format_for_issuance}
+## SD-JWT Issuance {#issued_sd-jwt}
```

Comment:
I propose to call this section something like simply "SD-JWT" or "SD-JWT Structure" and the to describe in one section both issuance and presentation. I would hope that the following structure works:

- An SD-JWT is structured like this: `<Issuer-signed JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure M>~<optional Key Binding JWT>`
- In issuance, no KB JWT, but all disclosures
- Here is an example for issuance:
- In presentation, maybe KB JWT, but only selected disclosures.
- Here is an example for presentation:

WDYT?

Reply:
> WDYT?

That was kinda what I was envisioning going into the editing but I got a little hesitant on making the bigger change of combing sections.  But I think what you're proposing is probably better so I'll take a pass at it.

Reply:
Tried to do that in 7ec2dba3b84418d55ecb12aabd18de6d7cdf6394